src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
You are linking a SCSS file in your HTML. However, browsers cannot interpret SCSS directly. You should compile your SCSS to CSS and link the CSS file instead.